### PR TITLE
Readme 1st draft

### PR DESCRIPTION
**Pull request by gonuke**
Partially satisfies #3 by proposing a top-level directory structure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+
+# Governance
+
+NECX is a community goverened effort according to a set of philosophies and
+policies to be fully developed in the near future. ([#20][i20])
+
+
+## Repository Layout
+
+* exchange: this is the repository of nodes, one per directory, named as the
```
Comment (katyhuff) [inline README.md]: maybe 'nodes' rather than exchange. either way works though.

**Review by katyhuff — APPROVED** [review]
LGTM

**Review by sskutnik — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
-# necx
\ No newline at end of file
+# NECX: Nuclear Engineering Curriculum eXchange
+
+The Nuclear Engineering Curriculum eXchange (NECX) is a open repository for
```
Comment (katyhuff) [inline README.md]: an open

**Comment by katyhuff** [comment]
bitmoji

File: README.md [hunk 3]
```diff
@@ -1 +1,39 @@
...
+* course notes including equations
+* example source code
+* citations of other work
+* external readings
+* instructor guidance
+* graphics
+* videos (or links to video content)
+* audio files (or links to audio content)
+* worked example problems
+* reference to ABET a-k objectives
```
Comment (TheDoctorRAB) [inline README.md:32]: We should add a link to the ABET web site.

**Pull request merged**